service-descriptions/link-example.json

```diff
@@ -193,26 +193,30 @@
   "components": {
     "links": {
       "UserRepositories": {
+        "name": "UserRepositories",
         "method": "get_repositories_by_owner",
         "params": {
           "username": "$result.username"
         }
       },
       "UserRepository": {
+        "name": "UserRepository",
         "method": "getRepository",
         "params": {
           "username": "$result.owner.username",
           "slug": "$result.slug"
         }
       },
       "RepositoryPullRequests": {
+        "name": "getPullRequestsByRepository",
```

Comment:
shouldnt this be `get_pull_requests_by_repository`

Reply:
maybe im confused between `name` and `method` here. i was expecting `name` to be method?